MangoAPI.Domain/Constants/ResponseMessageCodes.cs

```diff
@@ -54,6 +55,10 @@ public static class ResponseMessageCodes
         {Unauthorized, "User not authorized, please, sign in."},
         {InvalidEmailAddress, "Invalid email address. Please try another one."},
     };
+    
+    public static readonly ImmutableDictionary<string, string> ErrorDictionary = _dictionary.ToImmutableDictionary();
```

Comment:
use computable property here, not field

`public static ImmutableDictionary<string, string> ErrorDictionary =>_dictionary.ToImmutableDictionary();`